tensorflow/java/src/main/native/operation_jni.cc

```diff
@@ -120,10 +120,7 @@ JNIEXPORT jlongArray JNICALL Java_org_tensorflow_Operation_shape(
   std::unique_ptr<int64_t[]> cdims(new int64_t[num_dims]);
   TF_GraphGetTensorShape(graph, output, cdims.get(), static_cast<int>(num_dims),
                          status);
-  if (!throwExceptionIfNotOK(env, status)) {
-    TF_DeleteStatus(status);
-    return nullptr;
-  }
+                                        
```

Comment:
I don't think this change is correct. If `TF_GraphGetTensorShape` fails for any reason (and indicates the error by filling in `status`), with this PR, the error will be ignored.

Prior to this PR, we correctly convert the error to an exception.

Closing this PR since the change seems incorrect. If I've missed something, do let me know and we can reopen if required. Thanks.